packages/p2p-v2/src/components/TextArea/TextArea.tsx

```diff
@@ -11,6 +11,7 @@ type TTextAreaProps = HtmlHTMLAttributes<HTMLTextAreaElement> & {
     onChange: React.ChangeEventHandler<HTMLTextAreaElement>;
     placeholder?: string;
     shouldShowCounter?: boolean;
+    testId?: string;
     value?: string;
 };
 const TextArea = ({
```

Comment:
LGTM @adrienne-deriv, but I think this is going to be deleted soon, as the `deriv-com/ui` package has a `Textarea` component now 🤔

Reply:
The test cases can be updated once we replaced all the textarea components with deriv-com/ui